source/components/index.rst

```diff
@@ -1,15 +1,15 @@
-======================
+=========
```

Comment:
this length was used for a reason, why are you changing it?

also, this page was moved so it will need a redirect

Reply:
Seemingly an accident, although I don't see a technical reason that should matter.

If you mean a redirect from the root index.rst, that was added.